.idea/inspectionProfiles/profiles_settings.xml

```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
avoid adding `.idea` files to the commited files in the repo. You already added this folder to the `.gitignore` file, but I guess it was after adding these files

Reply:
good catch, thought github's .gitignore was already taking care of that